apps/web/src/common/modules/portals/HandbookButton.vue

```diff
@@ -84,16 +84,23 @@ export default defineComponent({
     setup(props, { emit }: SetupContext) {
         const state = reactive({
             proxyActiveTab: useProxyValue('activeTab', props, emit),
-            storageKey: computed<string>(() => `handbook:${store.state.user.userId}:${props.type}`),
+            userId: computed<string>(() => store.state.user.userId),
             noMore: false,
         });
 
-        watch(() => state.storageKey, () => {
-            state.noMore = !!LocalStorageAccessor.getItem(state.storageKey);
+        watch([() => state.userId, () => props.type], () => {
+            state.noMore = !!((LocalStorageAccessor.getItem(state.userId) ?? {}).handbook?.[props.type]);
```

Comment:
suggestion: I think this would be better!

```typescript
watch([() => state.userId, () => props.type], ([userId, type]) => {
            state.noMore = !!((LocalStorageAccessor.getItem(userId) ?? {}).handbook?.[type]);
```

Reply:
question: Is optional chaining not necessary here?
```suggestion
        watch([() => state.userId, () => props.type], () => {
            state.noMore = !!((LocalStorageAccessor.getItem(state.userId) ?? {})?.handbook?.[props.type]);
```